### ISSUE TITLE
Mapping between eye-gaze vectors and screen coordinates

### ISSUE THREAD

**Issue by BogdanBalcau** [TadasBaltrusaitis/OpenFace#974]
Is there any way to map the 3d gaze tracking data to specific points on screen?

**Comment by TadasBaltrusaitis** [comment]
No. there's no functionality to do that. See some ideas of how that could be implemented - https://github.com/TadasBaltrusaitis/OpenFace/issues/577

However, you would only get quite limited accuracy of mapping gaze from OpenFace to screen locations
